packages/TelegramClient-Tests.package/TCTMocks.class/class/mockSupergroupChat.st

```diff
@@ -0,0 +1,12 @@
+chat
+mockSupergroupChat
+
+	| chat |
+	chat := TCCChat new. 
+	chat id: 3; 
+		title: 'SupergroupMockChat'; 
+		type: 'chatTypeSupergroup'; 
+		lastMessage: 'lastSupergroupMessage'; 
+		canSendMessages: true; 
+		memberCount: 8972981. 
+	^chat 
```

Comment:
Is there something speaking against returning the Chat directly?
Like so:
```
	^ TCCChat new
	        chat id: 3;
		title: 'SupergroupMockChat'; 
		type: 'chatTypeSupergroup'; 
		lastMessage: 'lastSupergroupMessage'; 
		canSendMessages: true; 
		memberCount: 8972981. 
```
If you change it, please change it in `mockPrivateChat`, `mockBasicGroupChat`, `mockChats`, and `mockGroupChat` as well.